src/Utilities/MoistThermodynamics/src/MoistThermodynamics.jl

```diff
@@ -462,49 +784,212 @@ function saturation_adjustment(e_int, ρ, q_tot, T_init = T_triple)
     return air_temperature(e_int, q_tot, q_liq, q_ice)
   end
 end
+function saturation_adjustment_q_t_θ_l(θ_liq::DT, q_tot::DT, ρ::DT, p::DT) where DT
+  T_1 = θ_liq*(p/DT(MSLP))^(DT(R_d)/DT(cp_d))
```

Comment:
This is not consistent with our definition of dry potential temperature (which uses the moist-air gas constants. Replace by a function that inverts the liquid-ice potential temperature. Also, use liquid-ice potential temperature consistently, if at all. No need for a separate liquid potential temperature, I think.